pkg/etcd/etcd.go

```diff
@@ -451,7 +451,7 @@ func (e *ETCD) Start(ctx context.Context, clientAccessInfo *clientaccess.Info) e
 			select {
 			case <-time.After(30 * time.Second):
 				logrus.Infof("Waiting for agent to become ready before joining etcd cluster")
```

Comment:
should we do this too?
```suggestion
				logrus.Infof("Waiting for container runtime to become ready before joining etcd cluster")
```